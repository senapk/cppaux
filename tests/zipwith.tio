>>>>>>>>
========
[12, 14, 16, 18]
[111, 212, 313, 414]
[Aa, Bb, Cc, Dd, Ee, Ff]
[0p, 1t, 2e, 3r, 4o, 5d, 6a, 7c, 8t, 9i]
[0, 2, 4, 6, 8, 10, 12, 14, 16, 18]
<<<<<<<<
