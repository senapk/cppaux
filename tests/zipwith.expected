[12, 14, 16, 18]
[111, 212, 313, 414]
[Aa, Bb, Cc, Dd, Ee, Ff]
[0p, 1t, 2e, 3r, 4o, 5d, 6a, 7c, 8t, 9i]
